pkg/object/object.go

```diff
@@ -71,6 +205,24 @@ func (o *OpenCompose) Validate() error {
 		if err := service.Validate(); err != nil {
 			return err
 		}
+
+		// validate if the mounts are specified in root level volumes
+		// or emptydirvolumes
+		for _, container := range service.Containers {
+			for _, mount := range container.Mounts {
+				if !o.VolumeExists(mount.VolumeName) && !service.EmptyDirVolumeExists(mount.VolumeName) {
```

Comment:
There should be one more check that validates that there are not multiple mounts for same path.
```
    mounts:
    - volumeName: bar
      mountPath: /storage
    - volumeName: foo
      mountPath: /storage
```
This is not valid and should be checked.